openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -42,9 +41,6 @@ def handle(self, *args, **options):
             try:
                 course_keys = [CourseKey.from_string(arg) for arg in args]
             except InvalidKeyError:
-                log.fatal('Invalid key specified.')
-
-            if not course_keys:
-                log.fatal('No courses specified.')
```

Comment:
@dsjen In reading this code over with @robrap, we couldn't come up with an expected error case where this line would actually be executed.  So we've removed it from the code.  Please take one more look.  Thanks.